src/world/park.c

```diff
@@ -263,8 +263,8 @@ int calculate_park_rating()
 		result -= 100;
 
 		if (num_exciting_rides>0){
-			average_excitement = total_ride_excitement / num_exciting_rides;
-			average_intensity = total_ride_intensity / num_exciting_rides;
+			short average_excitement = total_ride_excitement / num_exciting_rides;
+			short average_intensity = total_ride_intensity / num_exciting_rides;
```

Comment:
I'd like to wait with that until this PR is merged, otherwise either of them will have a lot of conflicts.